tests/app/performance_test.py

```diff
@@ -0,0 +1,101 @@
+import subprocess
+import time
+import os
+import shutil
+import sys
+
+
+def get_cell(seconds):
+    cell = ', { "cell_type": "code", "execution_count": null, "metadata": {}, "outputs": [], ' \
```

Comment:
Wondering whether these cells could be created with `nbformat`?

Reply:
It seems a bit overkill to me.

Reply:
Wouldn't this help improve readability?

Reply:
It is true that it is not very readable. But do you mean using `nbformat` or Jinja?

Reply:
With `nbformat`, something similar to Voila's support for script files:

https://github.com/voila-dashboards/voila/blob/8557d9d270100e55a40911e1afe61046736eba18/voila/handler.py#L195-L204

And then `nbformat.write` to save the notebook file.

Reply:
I didn't know about that, I will look at it. Thanks!